### DISCUSSION TITLE
Group/association chaining

### DISCUSSION THREAD

**Discussion by zeev-mindali** [zwave-js/zwave-js-ui#3450] [Q&A]
Dear all,

when i making new group item, i can for example set nodeId 59 epId 1 on/off
but when i use another item to control it, the assosiation not working.

what i basicly means,
nodeId 59 epid 1 > nodeId60 epId 1 > nodeId 62 epId 2.
since i limitied to 5 assosiations, i want to set value of epid, and since it's also assosiated, it will control the other device.

**Comment by robertsLando** [comment]
@zeev-mindali It's not really clear to me what you are tring to do.

You want that node 59 on/off to be sent to node 60 that with the same association sends it to node 62? Why not simply add 2 associations on node 59 instead? 

Remember that associations works in this way:

Adding an association On/Off on node A to target node B means that the on/off of node A will be forwarded to node B (not vice versa)

**Comment by zeev-mindali** [comment]
since i limited to 5 assoiciations per device. 
if i can control device a which control device b which can control device c, i have multiplayed the number of associations.
for example if i want to close the entire home, if i can control other assoiciations, i can close the house with one button.
i can see the endpoint a  change the state on device b, but device b is assoisicated to device c, which not change it's state.

**Comment by robertsLando** [comment]
I dunno  if that's a protocol limitation, I mean an association that triggers another association could be disabled as it can create infinite loops (if you add an association on node b that triggers a that will do the loop).

@AlCalzone can you confirm my suppositions?

**Comment by AlCalzone** [comment]
Some devices support "forwarding" commands, but this is a device-specific implementation. If your devices don't specifically support that, you're out of luck.
